pom.xml

```diff
@@ -229,5 +185,11 @@ Changelog of Violation Comments to Bitbucket Server Plugin.
 			<version>2.5.0</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Parent POM started requiring upper bound dependency, this is in test scope. Jenkins is using an ancient one while PODAM is using a newer one. https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#231